<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab_main_window">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>841</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_team_session">
     <attribute name="title">
      <string>Team Session</string>
     </attribute>
     <widget class="QPushButton" name="btn_ts_create_session">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Create a session</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_ts_see_session">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>See sessions</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_ts_time_left_for_next_homework_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>50</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>the deadline for the next homework is at .....</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_ts_list_widget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Courses</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_list_view">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>321</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Homework details</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_ts_time_left_for_next_homework_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>90</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>the deadline for the next homework is at .....</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_ts_time_left_for_next_homework_4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>130</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>the deadline for the next homework is at .....</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_ts_time_left_for_next_homework_5">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>170</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>the deadline for the next homework is at .....</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_ts_time_left_for_next_homework_1">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>the deadline for the next homework is at .....</string>
      </property>
     </widget>
     <widget class="QListWidget" name="list_widget_ts_homework">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>771</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="list_widget_homework_detail">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>411</y>
        <width>761</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_time_management">
     <attribute name="title">
      <string>Time Management</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>350</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Pomodoro</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>410</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Pomodoro settings</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_tm_welcome">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Welcome</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_tm_course_task_date">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Course                          Task                     Date</string>
      </property>
     </widget>
     <widget class="QListWidget" name="list_widget_tm_example">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>256</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_tm_upcoming_events">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Upcoming events</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_material">
     <attribute name="title">
      <string>Materials</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_log_out">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>630</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>878</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
